.travis.yml

```diff
@@ -1,89 +1,94 @@
-        - python: "2.7"
-          env:
-            TOXENV=flake8
-        - python: "2.7"
-          env:
-            TOXENV=docs
-        - python: "2.7"
-          env:
-            TOXENV=locales
-            CREATE_DB=kuma
-            INSTALL_NPM_TOOLS=1
-        - python: "2.7"
-          env:
-            TOXENV=docker
-            INSTALL_DOCKER_COMPOSE=1
-            UID=0
-        - python: "2.7"
-          env:
-            TOXENV=assetlint
-            INSTALL_NPM_TOOLS=1
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
-    allow_failures:
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
+# matrix:
+#     include:
+#         - python: "2.7"
+#           env:
+#             TOXENV=pytest
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
+#         - python: "2.7"
+#           env:
+#             TOXENV=flake8
+#         - python: "2.7"
+#           env:
+#             TOXENV=docs
+#         - python: "2.7"
+#           env:
+#             TOXENV=locales
+#             CREATE_DB=kuma
+#             INSTALL_NPM_TOOLS=1
+#         - python: "2.7"
+#           env:
+#             TOXENV=docker
+#             INSTALL_DOCKER_COMPOSE=1
+#             UID=0
+#         - python: "2.7"
+#           env:
+#             TOXENV=assetlint
+#             INSTALL_NPM_TOOLS=1
+#         - python: "3.6"
+#           env:
+#             TOXENV=pytest3
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
+#     allow_failures:
+#         - python: "3.6"
+#           env:
+#             TOXENV=pytest3
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
 install:
-    - nvm install 10
-    - nvm use 10
-    - scripts/travis-install
-    - pip install -U pip
-    - pip install -r requirements/travis.txt
-    # Wait for ElasticSearch to be ready
-    - if [[ "$INSTALL_ELASTICSEARCH" == "1" ]]; then wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200; fi;
+#     - nvm install 10
+#     - nvm use 10
+    - INSTALL_DOCKER_COMPOSE=1 scripts/travis-install
+#     - pip install -U pip
+#     - pip install -r requirements/travis.txt
+#     # Wait for ElasticSearch to be ready
+#     - if [[ "$INSTALL_ELASTICSEARCH" == "1" ]]; then wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200; fi;
 script:
-    - tox -v
-after_failure:
-    - dmesg | tail
-notifications:
-    irc:
-        channels:
-            - "irc.mozilla.org#mdndev"
-        on_success: always
-        on_failure: always
-        use_notice: true
+      - docker-compose --version
+      - docker-compose pull
+      - docker-compose build
```

Comment:
We don't need to do this since we don't have any images that need to be built in the `docker-compose.yml` file.  Some time ago, we needed to do it for the `mysql` image I think, but no longer.

However, we do need to build a fresh `mdnwebdocs/kuma_base` Docker image. This is so the tests pick-up any changes made within the PR that would affect that image -- e.g., changes to the `docker/images/kuma_base/Dockerfile` itself of course, but also changes to Kuma's `package.json` or changes to many of the Python requirements files).

So I think instead of the `docker-compose build` we need something like `VERSION=latest make build-base` (which must be placed after the `docker-compose pull` so it overrides the pulled `mdnwebdocs/kuma_base`).

Reply:
I noticed that you're avoiding using the `make` commands, but if you do, you'll probably need to at least define `DJANGO_SETTINGS_MODULE=kuma.settings.testing` in the Travis environment.